balboa-agent/src/main/resources/config/config.properties

```diff
@@ -4,6 +4,12 @@
 # Current configuration for the Metric Consumer.
 balboa.agent.data.dir = /tmp
 balboa.agent.sleeptime = 10000
+balboa.agent.transport.type = MQ
```

Comment:
Could you comment this one out and put a comment above it indicating the two possible values and what they mean?